extensions/webchat/.eslintrc.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+module.exports = {
```

Comment:
this is quite the eslint file. is all this necessary?

Reply:
I had to do it as Extensions need to follow the same coding standards as Composer client. Since, extensions dont share tooling with Composer i had to create it as a copy of the eslint file from Composer core.